ampcrowd/basecrowd/urls.py

```diff
@@ -8,4 +8,6 @@
     url(r'^(\w+)/responses/$', views.post_response, name='post_response'),
     url(r'^(\w+)/tasks/$', views.create_task_group, name='create_tasks'),
     url(r'^(\w+)/purge_tasks/$', views.purge_tasks, name='purge_tasks'),
+    url(r'^(\w+)/summary/task_groups', views.get_task_groups, name='get_task_groups'),
+    url(r'^(\w+)/summary/tasks', views.get_tasks, name="get_tasks")
 )
```

Comment:
If it's too much hassle, I'm fine keeping it as one PR. But then we could definitely clean up the urls :smile: 

I'm not sure why the `/summary` prefix is necessary. Anything wrong with `/crowds/{crowd_name}/task_groups/` and `/crowds/{crowd_name}/tasks/`? Note the trailing slash, so we can support `/task_groups/{id}` when we want to click into one in the dashboard.